central/complianceoperator/v2/scanconfigurations/datastore/datastore_impl.go

```diff
@@ -186,3 +186,34 @@ func getScopeKeys(scanClusters []*storage.ComplianceOperatorScanConfigurationV2_
 
 	return clusterScopeKeys
 }
+
+func (ds *datastoreImpl) RemoveClusterFromScanConfig(ctx context.Context, clusterID string) error {
```

Comment:
We could add an e2e test for this in https://github.com/stackrox/stackrox/blob/master/tests/compliance_operator_v2_test.go so long as it's the last test and we keep processing these tests serially.

I imagine we'd need to have this be the last e2e test since I haven't seen any helper functions in our existing tests that re-add a cluster.

Reply:
I will create a ticket to add e2e test